backend/Application/Aggregates/Contract/Jobs/_10_CisEventReinitialization.cs

```diff
@@ -0,0 +1,174 @@
+using System.Threading;
+using System.Threading.Tasks;
+using System.Transactions;
+using Application.Aggregates.Contract.Configurations;
+using Application.Aggregates.Contract.Dto;
+using Application.Aggregates.Contract.Entities;
+using Application.Aggregates.Contract.EventLogs;
+using Application.Api.GraphQL;
+using Application.Api.GraphQL.EfCore;
+using Application.Api.GraphQL.Transactions;
+using Application.Resilience;
+using Dapper;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+
+namespace Application.Aggregates.Contract.Jobs;
+
+/// <summary>
+/// The job starts by truncate tables graphql_account_tokens, graphql_token_events and graphql_tokens. Reinitialization
+/// is needed both because mint events was not correctly updating account token balances.
+///
+/// For each contract those contract actions, which generates log events, are processed
+/// (contract initialization, contract interrupted and contract updated).
+///
+/// Each log events is checked if it should be parsed, see <see cref="CisEvent"/>. If the contract has a linked
+/// schema and a successfully human interpretable log event linked, the human interpretable log event is linked to
+/// the event. This may contain additional data.
+/// </summary>
+public sealed class _10_CisEventReinitialization : IStatelessJob
```

Comment:
Previously I just named them but it is now difficult to see, which job was created first.

This therefore starts enumeration of jobs by adding a prefix. Since some jobs already exist I start at 10 to start at a round number.